config/locales/en.yml

```diff
@@ -501,7 +501,7 @@ en:
         wrc_feedback_requested: ""
         wrc_incidents: "List of numbers referencing any incidents that you would like the WRC's feedback on."
         wic_feedback_requested: ""
-        wic_incidents: "List of numbers referencing any incidents that you would like the WIC's feedback on."
+        wic_incidents: "List of numbers referencing any incidents that you would like the WIC's feedback on. \nIf the WIC incident is sensitive (including but not limited to crimes, harassment, assault, etc.), please exclude this from your report and email the WIC separately. For other WIC incidents, default to a brief, anonymized version of the incident and follow-up with the WIC privately with further details. The WIC will still provide feedback on anonymized versions for general awareness. For all WIC incidents involving a WCA Volunteer, please exclude all details from the report, and contact the WIC privately."
```

Comment:
Please don't use `\n` in localized strings. If anything, you should probably use `<br/>`, but I'm not sure whether a line break is crucial here.

Reply:
Also, if you do end up using HTML tags inside the string, you need to rename it to `wic_incidents_html` so that Rails renders it correctly (if a translation key does not end with `_html`, then it will print the tag as if it were simple plain text)

Reply:
Also, the new parahrapgh that you're adding here is exactly the same as in the JS alert box above. If you move this message into a separate translation key, you could inject it into the HTML/JS above.

Reply:
Also, I'm not sure whether it's necessary to have the message as part of the alert and as part of the bottom description. If you really want both, you can introduce the alert text as a separate translation and then reference it here: `List of numbers referencing any incidents that you would like the WIC's feedback on. <strong>%{wic_incident_alert}</strong>`

If you do this, then the automated "guess the correct key" mechanism for `simple_form` won't work anymore, because you have to add the `wic_incident_alert` yourself. You can easily override any form hint like so: `<%= f.input :some_field, hint: I18n.t('my.custom.key') %>`